ci/test_wheel.sh

```diff
@@ -18,9 +18,31 @@ rapids-logger "Installing test dependencies"
 # echo to expand wildcard
 rapids-pip-retry install -v --prefer-binary -r /tmp/requirements-test.txt "$(echo "${DASK_CUDA_WHEELHOUSE}"/dask_cuda*.whl)"
 
+EXITCODE=0
+# shellcheck disable=SC2317
+set_exit_code() {
+    EXITCODE=$?
+    rapids-logger "Test failed with error ${EXITCODE}"
+}
+trap set_exit_code ERR
+set +e
```

Comment:
Along with lines 47-48, this reproduces the same error handling mechanism as that of `ci/test_python.sh`.